### PR TITLE
{vis}[fosscuda/2018b] Pango v1.42.4

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)
Depends on ~~#7653~~

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/869deed82bb0eaf9539cc53f7ca776c7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3108.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/8a385ea92ff3c947f5e9dc9bb6afb1dd for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2402.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/29777e4bcbed2e974ec7dd80209be2a2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2119.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c7e651b2bd9d4e7449d39958975364f0 for a full test report.

**Comment by boegel** [comment]
kicking Travis...

File: easybuild/easyconfigs/p/Pango/Pango-1.42.4-fosscuda-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+builddependencies = [
+    ('GObject-Introspection', '1.54.1', '-Python-2.7.15'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('X11', '20180604'),
+    ('GLib', '2.54.3'),
+    ('cairo', '1.14.12'),
+    ('HarfBuzz', '1.9.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/p/Pango/Pango-1.42.4-fosscuda-2018b.eb]: @akesandgren Any specific reason you used this old `HarfBuzz` version (see also failing test)?

Comment (akesandgren) [reply easybuild/easyconfigs/p/Pango/Pango-1.42.4-fosscuda-2018b.eb]: I don't remember. I think there was something I was building that didn't work with HarfBuzz 2 but right now I can't find it.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2402.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d18ea3db5e7f50d1262086ebae6e787b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2119.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ad3ba4a0855a65ec7668eedb382aa80a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3108.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/68651651bf9bff740bbb8eca4e9b9441 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**